docs/tutorials/density_maps.md

```diff
@@ -1,8 +1,6 @@
 # Density maps
 
-**Density maps** are a new concept introduced in QuPath v0.3.0.
-
-They were originally created to help find 'hotspots' for specific applications in pathology (e.g. tumor budding, Ki67 scoring).
+Density maps were originally created to help find 'hotspots' for specific applications in pathology (e.g. tumor budding, Ki67 scoring).
```

Comment:
Suggest

> Density maps were introduced in QuPath v0.3.0. They were originally...

Some people reading the docs may be reluctant upgraders, so I think it helps to retain references to when something was added (but ideally with better wording than mine, since mine doesn't date well :)